src/test/scala/edu/berkeley/cs/rise/opaque/OpaqueOperatorTests.scala

```diff
@@ -879,6 +880,30 @@ trait OpaqueOperatorTests extends OpaqueTestsBase { self =>
     KMeans.train(spark, securityLevel, numPartitions, 10, 2, 3, 0.01).map(_.toSeq).sorted
   }
 
+  testAgainstSpark("encrypted literal") { securityLevel =>
+    val input = 10
+    val enc_str = Utils.encryptScalar(input, IntegerType)
+
+    val data = for (i <- 0 until 256) yield (i, abc(i), 1)
+    val words = makeDF(data, securityLevel, "id", "word", "count")
+    val df = words.filter($"id" < decrypt(lit(enc_str), IntegerType)).sort($"id")
+    df.collect()
```

Comment:
```suggestion
    df.collect
```
Just to keep consistent style